services/ui-src/src/components/app/App.tsx

```diff
@@ -49,7 +51,8 @@ export const App = () => {
           <ReportProvider>
             <Timeout />
             <MainSkipNav />
-            <Header handleLogout={logout} />
+            {!isExportPage && <Header handleLogout={logout} />}
+            {isExportPage && <ExportedReportBanner />}
```

Comment:
suggestion: since these rely on each side of the same boolean you could write it as a ternary. I'm fine with this too.